[2022-01-25 10:37:14,906] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T10:36:56.507517+00:00 [queued]>
[2022-01-25 10:37:14,971] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T10:36:56.507517+00:00 [queued]>
[2022-01-25 10:37:14,977] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 10:37:14,978] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 10:37:14,980] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 10:37:15,026] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): load_data> on 2022-01-25T10:36:56.507517+00:00
[2022-01-25 10:37:15,047] {standard_task_runner.py:52} INFO - Started process 45 to run task
[2022-01-25 10:37:15,058] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'load_data', '2022-01-25T10:36:56.507517+00:00', '--job-id', '86', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmppqgf013f', '--error-file', '/tmp/tmptu_4ex0x']
[2022-01-25 10:37:15,068] {standard_task_runner.py:77} INFO - Job 86: Subtask load_data
[2022-01-25 10:37:15,210] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.load_data 2022-01-25T10:36:56.507517+00:00 [running]> on host 327d6291ad28
[2022-01-25 10:37:15,406] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T10:36:56.507517+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-25T10:36:56.507517+00:00
[2022-01-25 10:38:03,255] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 10:38:03,381] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=load_data, execution_date=20220125T103656, start_date=20220125T103714, end_date=20220125T103803
[2022-01-25 10:38:03,580] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-25 10:38:03,628] {local_task_job.py:146} INFO - Task exited with return code 0
